src/store/accounts/actions.js

```diff
@@ -219,7 +222,21 @@ export const verifyOTP = async function ({ commit, dispatch, state, rootState },
     }
   })
 
-  await this.$api.signTransaction(actions)
+  let options = {}
```

Comment:
@evgenibir we should have this as default value in one place and then reuse it instead of copy / pasting it

Reply:
fixed